src/pages/posts/[slug].astro

```diff
@@ -31,6 +31,6 @@ const { Content } = await post.render();
     </header>
 
     <Content />
-    <Utterances/>
+    <Utterances />
```

Comment:
I guess this was the auto formatter. should I remove it?

Reply:
Actually, I guess it matches the general formatting.

Reply:
Yeah, I like that it matches the general formatting to. I think that's the standard anyway?